libs/sdk-ui-filters/src/DateFilter/DateRangePicker/EndDateInputField.tsx

```diff
@@ -36,15 +42,29 @@ export const EndDateInputField = forwardRef<HTMLInputElement, IDateRangeInputFie
                     dateInputHintId: DATE_INPUT_HINT_ID,
                     timeInputHintId: TIME_INPUT_HINT_ID,
                 }}
-                onChange={onChange}
-                onInputMarkedValid={onInputMarkedValid}
-                value={value}
                 dateFormat={dateFormat}
                 isMobile={isMobile}
-                handleDayClick={onInputClick}
+                onDayClick={onInputClick}
                 isTimeEnabled={isTimeEnabled}
                 className={cx("s-date-range-picker-to", "gd-date-range-picker-to")}
                 defaultTime={DAY_END_TIME}
+                inputErrorMessageTexts={{
```

Comment:
place it to variable for better readability

Reply:
Fixed, also for the other blocks in the file.